src/Libraries/Microsoft.Extensions.Caching.Hybrid/Internal/DefaultHybridCache.MutableCacheItem.cs

```diff
@@ -21,16 +25,27 @@ public void SetValue(ref BufferChunk buffer, IHybridCacheSerializer<T> serialize
             buffer = default; // we're taking over the lifetime; the caller no longer has it!
         }
 
-        public override bool TryGetValue(out T value)
+        public void SetValue(T fallbackValue)
+        {
+            _fallbackValue = fallbackValue;
+        }
+
+        public override bool TryGetValue(ILogger log, out T value)
         {
             // only if we haven't already burned
             if (TryReserve())
             {
                 try
                 {
-                    value = _serializer.Deserialize(_buffer.AsSequence());
+                    var serializer = _serializer;
+                    value = serializer is null ? _fallbackValue! : serializer.Deserialize(_buffer.AsSequence());
```

Comment:
true, but as you say: I'm not convinced either is automatically "better"; in this case, I'm happy that having two lines reduces cognitive load, rather than trying to be clever with a one-liner (although disclosure: I'm guilty of those at other times; I'm absolutely an inconsistent hypocrite)